gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -186,15 +186,15 @@ def _shard_variants(known_args, pipeline_args, pipeline_mode):
   with beam.Pipeline(options=options) as p:
     variants = _read_variants(
         known_args.all_patterns, p, known_args, pipeline_mode)
-    call_names = (variants
-                  | 'CombineCallNames' >>
-                  combine_call_names.CallNamesCombiner())
+    call_sample_ids = (variants
```

Comment:
Again here it make sense to me to use `sample_ids`.

Reply:
Done.